src/app/views/jobs/jobs.component.ts

```diff
@@ -27,19 +36,41 @@ export class JobsComponent extends TranslationListener implements OnInit {
 
   ngOnInit() {
     this.loadData();
+
+    this.initLocation();
   }
 
   loadData() {
-    this.jobProxy.getJobs({include: 'owner,company,hourly_pay,company.company_images', 'filter[filled]': false, 'page[number]': this.page.toString()})
+    this.jobProxy.getJobs({ include: 'owner,company,hourly_pay,company.company_images', 'filter[filled]': false, 'page[number]': this.page.toString() })
       .then(result => {
         this.jobs = result.data;
         this.totalJobs = result.total;
       });
   }
 
+  initLocation() {
+    this.geolocationService.getLocation().subscribe(
+      position => {
+        this.mapUserLocation = new MapLocation({
+          longitude: position.coords.longitude,
+          latitude: position.coords.latitude
+        });
+        this.mapLocation = this.mapUserLocation;
+        this.mapZoom = 12;
+      },
+      error => {
+        console.log(error);
```

Comment:
Should we display some message to the user or something if an error occurs here?